lib/js/jsconsole.nim

```diff
@@ -25,45 +25,69 @@ proc debugImpl(console: Console) {.importcpp: "debug", varargs.}
 proc infoImpl(console: Console) {.importcpp: "info", varargs.}
 proc errorImpl(console: Console) {.importcpp: "error", varargs.}
 proc warnImpl(console: Console) {.importcpp: "warn", varargs.}
+proc traceImpl(console: Console) {.importcpp: "trace", varargs.}
 
 proc makeConsoleCall(console: NimNode, procName: NimNode, args: NimNode): NimNode =
   result = newCall(procName, console)
   for c in args: result.add(c)
 
 macro log*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/log
   makeConsoleCall(console, bindSym "logImpl", args)
 
 macro debug*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/debug
   makeConsoleCall(console, bindSym "debugImpl", args)
 
 macro info*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/info
   makeConsoleCall(console, bindSym "infoImpl", args)
 
 macro error*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/error
   makeConsoleCall(console, bindSym "errorImpl", args)
 
+macro exception*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/error
+  makeConsoleCall(console, bindSym "errorImpl", args)
```

Comment:
this seems fishy

Reply:
oh i see. maybe add a comment: 
> alias for `error` 

(from looking at https://developer.mozilla.org/en-US/docs/Web/API/Console/error)

also, alias'es should use templates

Reply:
`console.exception(21)` doesn't work for me:
Error: attempting to call undeclared routine: 'exception'

even if i use a template